dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
```

Comment:
The `convertConstructor` doesn't work for old school classes which inherit other classes. Moreover, if the `Constructor` has some non prototype properties then they are not copied over either.

Basically, the two new comprehensive `'stampit.convertConstructor()'` tests fail.